api/helpers/ find-spectatable-games.js

```diff
@@ -12,7 +12,11 @@ module.exports = {
       const games = await Game.find({
         status:  gameService.GameStatus.STARTED,
         updatedAt: { '>=': recentUpdateThreshhold },
-      });
+      })
+          .populate('players')
+          .map(({username, pNum}) =>{
+            return { username, pNum};
```

Comment:
I dont think we need the return keyword here, I believe you can just `.map(({username, pNum}) => ({username, pNum}))`

Reply:
Yes. I added that because I needed to put some consoles. I am going to remove it